### PR TITLE
test: init cypress and create smoke tests

### PR DESCRIPTION
**Pull request by dogivalfer**
# Description

Initialized Cypress with all the configuration needed and a few smoke tests to validate the config.

Solves # [FSADT2-46](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-46)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

Tested locally with the application pointing to the test environment.

## Checklist

- [ x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes
- [ x] Any dependent changes have already been accepted and merged

### CODE REVIEW

File: cypress/e2e/smoke-test/user-page.cy.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    // Delete user and verify that is deleted in the API
+    cy.contains('td', testUser.firstName).parent('tr').within(() => {
+      cy.get('td').eq(2).contains('button', 'Delete me').click()
+    });
+    cy.intercept('DELETE', `${Cypress.env('apiUrl')}/users/${testUser.firstName}/${testUser.lastName}`).as('delUser');
+    cy.wait('@delUser').its('response.statusCode').should('eq', 200);
+
+  });
+});
```
Comment (RMCampos) [inline cypress/e2e/smoke-test/user-page.cy.ts:53]: Same here. Few check style issues :/

**Review by RMCampos — CHANGES_REQUESTED** [review]
Just a few lint issues. Great job!! 🥳

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		body: {
+			firstName: firstname,
+			lastName: lastname
+		}
+	}).then((response) => {
+		console.log(response.status);
+		console.log(response.body);
+	});
+});
+
```
[the remark this answers is not in the document]
Comment (dogivalfer) [reply cypress/support/commands.ts:16]: Fixed all the lint problems, thanks for the catch! :+1:

**Review by RMCampos — APPROVED** [review]
LGTM. Good job!! 💯

**Pull request merged**